app/controllers/index.rb

```diff
@@ -0,0 +1,28 @@
+require 'sinatra'
+
+get '/' do
+  'Sealions'
+end
+
+#Create
+post '/sealions' do
+  Sealion.create(:name => params[:name])
+  redirect '/'
+end
+
+#Read
+get '/sealions/:id' do
+  "Sealion"
+end
+
+#Update
+put '/sealions/:id' do
+  Sealion.update_attribute(:name => params[:name])
```

Comment:
This will not ever update a sealion. update_attribute is an instance method, not a class method.